adr/adr-001.md

```diff
@@ -0,0 +1,28 @@
+# ADR-001: Exempt delegations as a transitionary mechanism for safe adoption of liquid staking.
+
+## Context 
+
+Liquid proof of stake may exacerbates the principal agent risk that exists at the heart of the delegated proof of stake system. The core of the problem is that validators do not actually own the stake that is delegated to them. This leaves the open to perverse incentives to attack the consensus system.
+
+Cosmos introduced the idea of min self bond in the staking. This creates a minimum amount of stake the must be bonded by the validators operator key. 
+
+This feature has very little effect on the behavior of delegates.
+
+## Proposal:
+
+The liquid staking system will introduce a new system. Delegator can issue a transaction that exempts their delegation to a specific validator from liquid staking. Once this transaction is issued, the SharesToToken tx will fail for the delegation. 
+
+Furthermore there will be a governance controlled parameter  that will restrict the total number of tokenized shares issued for each validator by the sum of the exempt delegations multiplied by a scalar value.
+
+For example, if governance has set value to 10.  Once a delegation exempts 1000 stake from the liquid staking system, then then other delegators may chose to issue up 10,000 tokenized shares with the tokens to shares transactions.
+
+If the maximum amount of tokenized shares are issued, the exempt delegation cannot be undelegated unless the validator chooses to leave the active validator set.
+ 
+
+## Economics:
```

Comment:
My comment is based on the following interpretation/ example of the ADR:
- A delegator (called A) can exempt part of his tokens from liquid staking. E.g., a delegator has 100,000 tokens, and he exempts 1,000.
- Another delegator (B) in the same validator also has 100,000 tokens and exempts 1,000.
- There is a multiplicative parameter>1 that states an aggregate limit in the number of tokens that can be made liquid from such validator. Following the ADR, let suppose this parameter is 10.
- The aggregate limit is 20,000. The delegators (and the validator in case she has her self-staked tokens) can make liquid up to 20,000. 
- There is no rule of how this 20,000 limit is distributed between the 3 participants mentioned above.

Comments:
- Clearly, if no delegator exempts part of his token, nobody can participate in the liquid staking. So, there are incentives to make the aggregate exemption different from zero.
- It seems to be incentives for free-ride. That is, if C knows that A and B are going to exempt 2,000 (and hence impose a limit of 20,000), the new delegator might decide not to exempt. Or to wait for what happens and see if the other delegators do not cover the limit of 20,000.
- If the multiplicative parameter is not very high, these free-riding incentives seem to decrease. That is because there is less room for underutilization of the limit (in the example, A and B do not cover 20,000) 
- Alternatively, to avoid free-riding, it is possible to design a limit per-delegator who is exempting (additionally to the
aggregate limit. In the previous example, A could have a limit of 10,000 (e.g., 50%) and B the remaining 10,000. Thus, a no contributing delegator cannot participate in the liquid staking.
- With this new limit, the delegators compete in increasing their share of the aggregate limit by exempting more tokens, increasing Cosmos security.
- It also solves the issue of how to distribute the limit between the different participants mentioned above.
- Alternatively, there could be a protocol that (exogenously) imposes the number of tokens that are exempted for each delegator. That might simplify some aspects of the proposal, but the delegators will lose the ability to decide how much to exempt based on their risk preferences.